polish-legal.csl

```diff
@@ -101,10 +102,20 @@
   </macro>
   <macro name="locator">
     <!-- Numer strony. -->
-    <group delimiter=" ">
-      <label variable="locator" form="short" suffix="."/>
-      <text variable="locator"/>
-    </group>
+    <choose>
+      <if locator="section">
+        <group delimiter=" ">
+          <label variable="locator" form="long"/>
+          <text variable="locator"/>
+        </group>
+      </if>
+      <else>
+        <group delimiter=" ">
+          <label variable="locator" form="short" suffix="."/>
```

Comment:
the  suffix="." isn't really needed here, is it? The periods are already in the abbreviations. If that's right, we could just define teza as the short form of section above and the wouldn't have to have a separate if-else here but 
```
       <group delimiter=" ">
          <label variable="locator" form="short"/>
          <text variable="locator"/>
        </group>
```

would just work for everything.

Reply:
I checked your proposal, works fine. Thank you!